guide/java/archetype.md

```diff
@@ -26,11 +26,13 @@ Alternatively, all options can be supplied at the command line. For example,
 if creating a project named "autobrick" for "com.acme":
 
 {% highlight bash %}
+# BROOKLYN_VERSION={{ site.brooklyn-version }}
 $ mvn archetype:generate \
 	-DarchetypeGroupId=org.apache.brooklyn \
 	-DarchetypeArtifactId=brooklyn-archetype-quickstart \
-	-DarchetypeVersion={{ site.brooklyn-version }} \
-	-DgroupId=com.acme -DartifactId=autobrick \
+	-DarchetypeVersion=${BROOKLYN_VERSION} \
```

Comment:
Changing to `$ BROOKLYN_VERSION={{ site.brooklyn-version }}` at the start (I didn't mean it to be a comment!). Hopefully having those as two separate lines will still let someone copy-paste.